cli/towerSysUtil.ml

```diff
@@ -41,6 +41,11 @@ let buffer in_file =
   read_file (fun bytes len -> Buffer.add_subbytes buf bytes 0 len) in_file;
   buf
 
+let ignorable_file file_name =
+  (* Currently hard-codes any file starting with . or _ as ignorable by Tower *)
+  let ignorable_pattern = (Str.regexp "^[_\\.]") in
+  Str.string_match ignorable_pattern file_name 0;;
```

Comment:
This seems like a pretty heavy way to check the first character of a string. I'd rather just use `String.get file_name 0` until we definitely know we need full regexes. Also, this prevents the `str` dep.